Neos.Flow/Configuration/Settings.Http.yaml

```
@@ -24,9 +24,15 @@ Neos:
       # request (default).
       baseUri: NULL
 
+      # Defines the PSR-15 middlewares that a request goes through in a flow
+      middlewares:
+        'dispatch':
```

Note: This is called `dispatch` because in the future we will end up with an actual `MvcDispatchMiddleware` here, so other middlewares configured in relative position to this one won't have to change.

---

i would be fine with calling it `legacyHttpChain` now and remove it with Flow 7

---

We could do this, but then again at some point we will need some "final middleware" that creates the response, which IMO is our MVC dispatch. So I'd be in favor of finding a good appropriate name for this future single middleware right now.
`mvcdispatch`/`mvc-dispatch` could be more exact

---

In the end both would work, i prefer to not name things after future roles as those assumptions may change.

there is no harm if we end up with 

```
middlewares:
   dispatch:
        middleware: SomeClass
   legacyHttpChain: null
```

---

In general I'm with you regarding "assumptions may change". But:
- that we will have an "MVC dispatch" at some point is pretty much a given, unless we're talking about Flow 20 or s.th. where probably a new approach is the base for the framework ;)
- I do think that renaming the middleware config does some harm, because this configuration is based on sorted arrays, which are relative to other *names* - so if someone will configure his own middleware as `before legacyHttpChain` now, he will need to change that later on. Not world-shaking harm, but something we could prevent *most likely* now

```suggestion
        'mvc-dispatch':
```